classes/patterns/class-action.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Action pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Action.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Action extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/action-pattern-layout';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 *
+	 * @param array $params Optional array of parameters for the config.
+	 */
+	public static function get_config( $params = [] ): array {
+		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
+
+		return [
+			'title'      => __( 'Action', 'planet4-blocks-backend' ),
+			'categories' => [ 'layouts' ],
+			'content'    => '
+				<!-- wp:group {"className":"block"} -->
+					<div class="wp-block-group">
+						<!-- wp:media-text {"mediaLink":"' . $media_link . '","mediaType":"image"} -->
+							<div class="wp-block-media-text is-stacked-on-mobile">
+								<figure class="wp-block-media-text__media">
+									<img src="' . $media_link . '" alt="' . __( 'Default image', 'planet4-blocks-backend' ) . '"/>
+								</figure>
+								<div class="wp-block-media-text__content">
+									<!-- wp:gravityforms/form /-->
+								</div>
+							</div>
+						<!-- /wp:media-text -->
+						<!-- wp:group {"backgroundColor":"grey-05","align":"full","className":"is-style-large-padding"} -->
+							<div class="wp-block-group alignfull has-grey-05-background-color has-background is-style-large-padding">
+								<!-- wp:group {"className":"container"} -->
+									<div class="wp-block-group container">
+										' . SideImageWithTextAndCta::get_config(
+										[
+											'media_position'    => 'right',
+											'title_placeholder' => __( 'The problem', 'planet4-blocks' ),
```

Comment:
Why did you define the domain to `planet4-blocks` but from [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/879/files#diff-ff56a7b10430fcb2a283a3994a55fad63d259fb799ad2539f4da4f42798a16e6R42) you defined to `planet4-blocks-backend`?

Reply:
I've left `planet4-blocks` here because it could still appear in the frontend in case editors don't change it